celery_with_django/django_celery_project/settings.py

```diff
@@ -0,0 +1,172 @@
+"""
+Django settings for django_celery_project project.
+
+Generated by 'django-admin startproject' using Django 5.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
```

Comment:
Install the import optimizer plugin @shimpidipali I will help you out

Reply:
ok